Gordon360/Services/NewsService.cs

```diff
@@ -283,6 +283,12 @@ public string GetImagePath(string filename)
             return Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", "news", filename);
         }
 
+        private string GetImageURL(string serverAddress, string filename)
+        {
+            var url = $"{serverAddress}/browseable/uploads/news/{filename}";
```

Comment:
Since this is an instance method, you can access `_serverUtils` and the server address from within the function. Then you can remove that take that logic out of the other methods.
```suggestion
        private string GetImageURL(string filename)
        {
            var serverAddress = _serverUtils.GetAddress();
            if (serverAddress is not string) throw new Exception("Could not upload Student News Image: Server Address is null");
            var url = $"{serverAddress}/browseable/uploads/news/{filename}";
```